### PR TITLE
Base 20.00.07.00

### PR DESCRIPTION
**Pull request by tdclienttools**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by tdclienttools** [comment]
> @tdclienttools Please fix the PR to only include the commits that are relevant to the changes being made here, and do not use merge commits.

We removed the commits which are not relevant to this pull request. 
We also did the rebase.

**Comment by bevanjkay** [comment]
There are still two commits that are unrelated.

**Comment by tdclienttools** [comment]
> There are still two commits that are unrelated.

Are you referring to the last two commits?

**Comment by tdclienttools** [comment]
Re: There are still two commits that are unrelated.
We responded to the above comment.

File: Casks/t/ttu-base-suite.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
   installer script: {
     executable: "silent-install.sh",
-    args:       ["ttu-base-suite-2000"],
+    args:       ["ARGS"],
```
Comment (bevanjkay) [inline Casks/t/ttu-base-suite.rb:22]: @tdclienttools can you explain why this change is being made?

Comment (tdclienttools) [reply Casks/t/ttu-base-suite.rb:22]: The "ARGS" should have been removed since no parameters are required for the silent-install.sh. In the previous submission the tag 'ARGS' was replaced with 'ttu-base-suite-2000'. In this submission we did not replace the TAG.

Comment (bevanjkay) [reply Casks/t/ttu-base-suite.rb:22]: So if I am understanding correctly, instead of changing the contents of the args array, the entire `args` parameter should be removed?

Comment (bevanjkay) [inline Casks/t/ttu-base-suite.rb:22]:
```suggestion
```

**Review by bevanjkay — APPROVED** [review]
